scripts/qgis-cmake-setup.sh

```diff
@@ -48,9 +48,8 @@ if ! [[ "$INSTALL_DIR" = /* ]] || ! [ -d "$INSTALL_DIR" ]; then
   usage
 fi
 
-CMAKE=$(which cmake)
-if [ -z $CMAKE ]; then
-  echo "CMake executable 'cmake' not found"
+if ! (command -v cmake >/dev/null 2>&1); then
```

Comment:
Hi @brncsk, thanks for the contribution!

Curious, why use that construct for finding `cmake` and `ninja` instead of e.g. `if ! (which -s cmake); then`?